src/test/java/org/jabref/logic/bibtex/comparator/BibStringDiffTest.java

```diff
@@ -0,0 +1,39 @@
+package org.jabref.logic.bibtex.comparator;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibtexString;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class BibStringDiffTest {
+
+    private final BibDatabase originalDataBase = mock(BibDatabase.class);
+    private final BibDatabase newDataBase = mock(BibDatabase.class);
+    private final BibtexString str1 = new BibtexString("name", "content");
+    private final BibtexString str2 = new BibtexString("name2", "content2");
+    private final BibtexString str3 = new BibtexString("name2", "content3");
+    private final BibStringDiff diff = new BibStringDiff(str2, str3);
+
+    @BeforeEach
+    void setUp() {
+        when(originalDataBase.hasNoStrings()).thenReturn(false);
+        when(newDataBase.hasNoStrings()).thenReturn(false);
+        when(originalDataBase.getStringValues()).thenReturn(Arrays.asList(str1, str2));
+        when(newDataBase.getStringValues()).thenReturn(Arrays.asList(str1, str3));
```

Comment:
Inline the variables of above.

```suggestion
        when(originalDataBase.getStringValues()).thenReturn(List.of(new BibtexString("name", "content"), new BibtexString("name", "content2"));
        when(newDataBase.getStringValues()).thenReturn(List.of(new BibtexString("name", "content"), new BibtexString("name", "content3"));
```